fmriprep/workflows/anatomical.py

```diff
@@ -75,10 +79,46 @@ def t1w_preprocessing(name='t1w_preprocessing', settings=None):
         ),
         name='T1_2_MNI_Registration'
     )
-    # should not be necesssary byt does not hurt - make sure the multiproc
+    # should not be necesssary but does not hurt - make sure the multiproc
     # scheduler knows the resource limits
     t1_2_mni.interface.num_threads = settings['ants_nthreads']
 
+    # 6. FreeSurfer reconstruction
+    autorecon1 = pe.Node(
+        freesurfer.ReconAll(
+            directive='autorecon1',
+            args='-noskullstrip -parallel -openmp {:d}'.format(cpu_count())),
+        name='Reconstruction')
+    autorecon1._interface._can_resume = False
+    if 'FREESURFER_SUBJECTS' in os.environ:
+        autorecon1.inputs.subjects_dir = os.getenv('FREESURFER_SUBJECTS')
+
+    convert_asw = pe.Node(
+        freesurfer.MRIConvert(out_type='mgz'),
+        name='SkullstrippedMGZ')
+
+    def inject_skullstripped(subjects_dir, subject_id, skullstripped):
+        import os
+        from nipype.utils.filemanip import copyfile
+        mridir = os.path.join(subjects_dir, subject_id, 'mri')
+        bm_auto = os.path.join(mridir, 'brainmask.auto.mgz')
+        bm = os.path.join(mridir, 'brainmask.mgz')
+        copyfile(skullstripped, bm_auto)
+        copyfile(bm_auto, bm)
+        return subjects_dir, subject_id
+
+    injector = pe.Node(
+        niu.Function(
+            function=inject_skullstripped,
+            input_names=['subjects_dir', 'subject_id', 'skullstripped'],
+            output_names=['subjects_dir', 'subject_id']),
+        name='InjectSkullstrip')
+
+    reconall = pe.Node(
+        freesurfer.ReconAll(
+            args='-parallel -openmp {:d}'.format(cpu_count())),
+        name='Reconstruction2')
```

Comment:
Having an issue here:

    traits.trait_errors.TraitError: The trait 'subjects_dir' of a ReconAllInputSpec instance is an existing
    directory name, but the path '/scratch/workflow_enumerator/01/t1w_preprocessing/Reconstruction' does not exist.

But... it's still running `Reconstruction2`, and I'm getting outputs. Is this a nipype bug?